### DISCUSSION TITLE
Nutshell 🐿️ - an AI meeting assistant powered by node-llama-cpp

### DISCUSSION THREAD

**Discussion by NutshellAI** [withcatai/node-llama-cpp#382] [Show and tell]
Hi everyone! I’m excited to share **_Nutshell_**, a <ins>completely private</ins>, AI-powered transcription and meeting assistant application that leverages OpenAI’s Whisper model using [MLX](https://github.com/ml-explore/mlx-examples/tree/main/whisper) for real-time, on-device transcriptions and [node-llama-cpp](https://github.com/withcatai/node-llama-cpp) to power a fully contextual chat assistant. We've been working on it for the past couple of months and finally have a product that we're ready to share.

## **What is Nutshell?** 
Nutshell captures your meetings, transcribing audio and summarizing discussions, all while keeping your data secure on your device. This ensures full privacy and compliance with corporate data policies. 🎧✨

### Key Features:
- <ins>**Real-Time Transcription**</ins> powered by Whisper 
- <ins>**AI Meeting Assistant**</ins> that provides contextual summaries, action items, and Q&A
- <ins>**On-Device Processing**</ins> for enhanced privacy
- <ins>**Customizable Prompts**</ins> to tailor summaries and actions
- <ins>**Organizational Tools**</ins> with folders and export options for meeting transcriptions
- <ins>**No bots**</ins> ever sent to your meetings

**You can download and try it out on our website: [ withnutshell.com](https://withnutshell.com/). Let me know what you think, would love to hear your feedback!**

🙏 _A big thank you to the the llama.cpp community, we are incredibly grateful to this community of developers. Your shared knowledge, support, and collaborative spirit have been instrumental in our learning journey and growth while building Nutshell._

❤️ 🐿️ 
from the founders at Nutshell

![main_nutshell](https://github.com/user-attachments/assets/9996ec6e-993a-43d6-95dd-82ac1713e666)

![chat_assistant](https://github.com/user-attachments/assets/9a9a7fc2-2ce4-46f8-a43b-56c9e20cf4f0)

**Comment by giladgd** [comment]
Looks very nice!
If it's open-source, you can open a PR to add it to the [awesome list](https://node-llama-cpp.withcat.ai/guide/awesome).
